src/zope/server/http/tests/test_commonaccesslogger.py

```diff
@@ -105,14 +100,13 @@ def t():
         time.altzone = -3600
         time.time = t
         try:
-            output = Output()
-            cal = CommonAccessLogger(output, Resolver())
+            cal = CommonAccessLogger(resolver=Resolver())
             cal.log(Task())
         finally:
             time.daylight = orig_dl
             time.altzone = orig_az
             time.time = orig_t
 
         self.assertEqual(
-            'host - anonymous [29/Nov/1973:21:33:09 +0100] "GET / HTTP/1.0" 200 OK 10 "" ""\n',
-            output.msgs[0])
+            'accesslog INFO\n  host - anonymous [29/Nov/1973:21:33:09 +0100] "GET / HTTP/1.0" 200 OK 10 "" ""',
+            str(handler))
```

Comment:
This is mimicking the standard Apache access.log format, and it feels weird to see Python logging miscellanea in front of it.  Maybe set a logging.Formatter('%(message)s') in the test setUp to keep it plain?

(I'm possibly overthinking things again.)

Reply:
`str` of `InstalledHandler` doesn't go through logging formatters, instead formatting things itself. But that's a good point, this can be cleaned up.